x-pack/test/functional/config.js

```diff
@@ -384,6 +384,17 @@ export default async function ({ readConfigFile }) {
           },
         },
 
+        test_logs_data_reader: {
+          elasticsearch: {
+            indices: [
+              {
+                names: ['logs-*', 'test_data_stream'],
```

Comment:
I'm not sure I understand the privs required around data streams.  I'll do a little research around this before approving the PR

Reply:
removed the un-used `logs-*` index name